docs/src/embeddings/default_embedding_functions.md

```diff
@@ -186,21 +186,26 @@ We offer support for all huggingface models (which can be loaded via [transforme
 Example usage - 
 ```python
 import lancedb
-import pandas as pd
-
 from lancedb.embeddings import get_registry
 from lancedb.pydantic import LanceModel, Vector
 
+db = lancedb.connect("/tmp/db")
 model = get_registry().get("huggingface").create(name='facebook/bart-base')
 
-class TextModel(LanceModel):
+
+class Words(LanceModel):
     text: str = model.SourceField()
     vector: Vector(model.ndims()) = model.VectorField()
 
-df = pd.DataFrame({"text": ["hi hello sayonara", "goodbye world"]})
-table = db.create_table("greets", schema=Words)
-table.add()
-query = "old greeting"
+
+table = db.create_table("words", schema=Words)
+table.add(
+    [
+        {"text": "hello world"},
+        {"text": "goodbye world"}
+    ]
+)
+query = "greetings"
```

Comment:
both are greetings, we could show a more clear example. Either the older example or something simple yet distinct , for ex :

```python
table.add(
    [
        {"text": "Golden Retriever energy"},
        {"text": "Black cat"}
    ]
)
query = "playful dog"
```

Reply:
Thanks for the comment, happy to use this - would you like me to change all the other examples to match this, i.e.:

https://github.com/lancedb/lancedb/blob/fdb5d6fdf1454a1c545cc9da53abad8d1ccb6b99/docs/src/embeddings/default_embedding_functions.md?plain=1#L169-L176

https://github.com/lancedb/lancedb/blob/fdb5d6fdf1454a1c545cc9da53abad8d1ccb6b99/docs/src/embeddings/default_embedding_functions.md?plain=1#L202-L203

https://github.com/lancedb/lancedb/blob/fdb5d6fdf1454a1c545cc9da53abad8d1ccb6b99/docs/src/embeddings/default_embedding_functions.md?plain=1#L231-L238

https://github.com/lancedb/lancedb/blob/fdb5d6fdf1454a1c545cc9da53abad8d1ccb6b99/docs/src/embeddings/default_embedding_functions.md?plain=1#L287-L291

https://github.com/lancedb/lancedb/blob/fdb5d6fdf1454a1c545cc9da53abad8d1ccb6b99/docs/src/embeddings/default_embedding_functions.md?plain=1#L322-L327

https://github.com/lancedb/lancedb/blob/fdb5d6fdf1454a1c545cc9da53abad8d1ccb6b99/docs/src/embeddings/default_embedding_functions.md?plain=1#L372-L377

Reply:
yes sure wherever there is less clarity you can change there, thanks a lot !